File: 백준/Silver/20291. 파일 정리/파일 정리.py
from collections import defaultdict
import sys

input = sys.stdin.readline

n = int(input())
ext_map = defaultdict(int)
for _ in range(n):
    _, extension = input().strip().split('.')
    ext_map[extension] += 1

for ext in sorted(ext_map):
    print(f"{ext} {ext_map[ext]}")

File: 백준/Silver/20291. 파일 정리/파일 정리.kt
fun main() {
    val n = readln().toInt()
    val extMap = mutableMapOf<String, Int>()

    for (i in 1..n) {
        val (_, extension) = readln().split(".")
        extMap[extension] = extMap.getOrDefault(extension, 0) + 1
    }

    val sortedExtension = extMap.keys.sorted()
    for (ext in sortedExtension) {
        println("$ext ${extMap[ext]}")
    }
}